src/core/utils/chainProviders.ts

```diff
@@ -20,20 +19,21 @@ const provider = ({
 
   if (!enableMulticallQueue) {
     logger.debug({ chainId }, `Using standard provider`)
-    return new ethers.providers.StaticJsonRpcProvider(url, chainId)
+    return new ethers.JsonRpcProvider(url, chainId, {
+      staticNetwork: Network.from(chainId),
+    })
```

Comment:
There's no `StaticJsonRpcProvider`, the new way of using it is by passing a `staticNetwork` property to the settings.

Reply:
makes sense